### ISSUE TITLE
Merge some great NewPipe PRs

### ISSUE THREAD

**Issue by rodrigoswz** [polymorphicshade/NewPipe#49]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

Official NewPipe repo has several incredible contributions from the community, unfortunately many of them are ready and waiting for approval for even 2 years.

As it may take even longer to be approved (or not), would it be considerable merge them on that fork?

Some great examples:
Subscriptions shortcuts: https://github.com/TeamNewPipe/NewPipe/pull/2434

AutoBackup: https://github.com/TeamNewPipe/NewPipe/pull/3157

Add settings to match device's theme (dark & black): https://github.com/TeamNewPipe/NewPipe/pull/3632

#### Is your feature request related to a problem? Please describe it

No

#### How will you/everyone benefit from this feature?

These 3 PRs that I mentioned make perfect sense to be included in the app.

**Comment by MD77MD** [comment]
I vote up these PRs too:

Add support for tabs #3201
https://github.com/TeamNewPipe/NewPipe/pull/3201#issuecomment-851654049

Sleep timer #2892
https://github.com/TeamNewPipe/NewPipe/pull/2892

YouTube's Fast Forward/Rewind behavior #4833
https://github.com/TeamNewPipe/NewPipe/pull/4833

**Comment by login22rvp** [comment]
I agree it should more be called NewPipeX... Which include many experimental features keeping the core intact. People call choose to switch on and off basis their requirement. The core experience will remain same as main NewPipe.

NewPipeX can be a digital lab..

**Comment by Atemu** [comment]
Good luck with your fork @login22rvp!

**Comment by DUOLabs333** [comment]
Sounds interesting --- I would like to set it up.
